Use brackets around info variables (#14206)
netdata/netdata: health/REFERENCE.md

| variable | description |
| ---------| ----------- |
| ${family}  | Will be replaced by the family instance for the alert (e.g. eth0) |

> | ${label:XXX}  | This will replace the variable with the chart label's XXX value |

Reviewer: ```suggestion
| ${label:LABEL_NAME}  | The variable will be replaced with the value of the label |
```

I think we shouldn't use [XXX](https://www.dictionary.com/e/slang/xxx/).

Reviewer 2: Agreed, for multiple reasons.

Reviewer 3: I agree 100% with @Ferroin ! :)

Author: Done!!
